_includes/02-image.md

```diff
@@ -0,0 +1 @@
+Replace this with an image, like your photo. Ensure you include some alt-text.
```

Comment:
![Image of Yaktocat](https://octodex.github.com/images/yaktocat.png)
![68747470733a2f2f6f63746f6465782e6769746875622e636f6d2f696d616765732f79616b746f6361742e706e67](https://user-images.githubusercontent.com/81251742/120321446-f0111600-c300-11eb-9549-3c6951949f80.png)